stdlib/public/Concurrency/TaskCancellation.swift

```diff
@@ -25,9 +25,7 @@ extension Task {
   /// - SeeAlso: `checkCancellation()`
   /* @instantaneous */
   public static func isCancelled() async -> Bool {
-    // let task = __getTask() // TODO: pending internal API to get tasks
-    // task.isCancelled || task.deadline.isOverdue
-    fatalError("\(#function) not implemented yet.")
+     isTaskCancelled(Builtin.getCurrentAsyncTask())
```

Comment:
I assume this is an unfinished PR, apologies for the interjection then, just noticed inconsistent spacing

```suggestion
    isTaskCancelled(Builtin.getCurrentAsyncTask())
```

Reply:
It's a part of a bigger PR that I'm working on and extracting smaller commits out of it when able to, since it's nicer to review/keep moving this way.

Thanks, applied the fix